src/main/java/org/mate/bookstorespringboot/controller/BookController.java

```diff
@@ -0,0 +1,36 @@
+package org.mate.bookstorespringboot.controller;
+
+import java.util.List;
+import lombok.RequiredArgsConstructor;
+import org.mate.bookstorespringboot.controller.dto.BookDto;
+import org.mate.bookstorespringboot.controller.dto.BookRequestDto;
+import org.mate.bookstorespringboot.service.interfaces.BookService;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequiredArgsConstructor
+@RequestMapping("/api/books")
+public class BookController {
+
+    private final BookService bookService;
+
+    @GetMapping
+    public List<BookDto> getBooks() {
+        return bookService.findAll();
+    }
+
+    @GetMapping("/{id}")
+    public BookDto getBookById(@PathVariable Long id) {
+        return bookService.findById(id);
+    }
+
+    @PostMapping
```

Comment:
Let’s  add response status Created

Reply:
I will be using ResponseEntity in controller layers